RadarSDK/RadarTripOptions.m

```diff
@@ -77,6 +79,7 @@ - (NSDictionary *)dictionaryValue {
     dict[kDestinationGeofenceExternalId] = self.destinationGeofenceExternalId;
     dict[kMode] = [Radar stringForMode:self.mode];
     dict[kScheduledArrivalAt] = self.scheduledArrivalAt;
+    dict[kApproachingThreshold] = @(self.approachingThreshold);
```

Comment:
Need to add this to `isEqual` below

Reply:
durgh, had that in my local commit history. Just pushed up